### PR TITLE
Manage different newlines ('\r\n', '\n') with the Transcriber

### PR DESCRIPTION
**Pull request by kbairak**
@lolletsoc just an idea I had about handling different types of newline sequences (unix/dos) without really having to handle them.

### CODE REVIEW

File: openformats/transcribers.py [hunk 1]
```diff
@@ -214,7 +215,43 @@ def line_number(self):
         return self.newline_count + 1
 
     def get_destination(self):
-        return "".join([entry
+        return "".join([self.edit_newlines(entry)
```
Comment (lolletsoc) [inline openformats/transcribers.py]: You can remove the `[` and `]` from the list comprehension and turn it into a generated one

File: openformats/transcribers.py [hunk 2]
```diff
@@ -214,7 +215,43 @@ def line_number(self):
...
+        `get_destination()` will convert all newline sequences to the correct
+        ones.
+
+        This way, if a source file has DOS newline sequences, and you don't
+        want to handle this case within the logic of the parser::
+
+            >>> is_dos_type = "\r\n" in source
+            >>> if is_dos_type:
+            ...     source = source.replace("\r\n", "\n")
+            >>> transriber = Transcriber(source)
```
Comment (lolletsoc) [inline openformats/transcribers.py]: transriber -> transcriber

File: openformats/transcribers.py (hunk 1 above)
Comment (kbairak) [inline openformats/transcribers.py]: Good call, missed that one.

File: openformats/transcribers.py [hunk 3]
```diff
@@ -214,7 +215,43 @@ def line_number(self):
         return self.newline_count + 1
 
     def get_destination(self):
-        return "".join([entry
+        return "".join((self.edit_newlines(entry)
```
Comment (lolletsoc) [inline openformats/transcribers.py]: You don't need the `(` and `)` either!

**Comment by luzpaz** [comment]
bump

**Pull request merged**